rules/falco_rules.yaml

```diff
@@ -2604,7 +2604,25 @@
   output: "Docker or kubernetes client executed in container (user=%user.name %container.info parent=%proc.pname cmdline=%proc.cmdline)"
   priority: WARNING
   tags: [container, mitre_execution]
+
+
+# This rule is not enabled by default, as there are legitimate use
+# cases for raw packet. If you want to enable it, modify the
+# following macro.
+- macro: consider_raw_packet_communication
+  condition: (never_true)
+
+- list: user_known_raw_packet_binaries
+  items: []
+
+- rule: Raw packet created in container
+  desc: Detect new raw packets at the device driver (OSI Layer 2) level in a container. raw packets could be used to do ARP Spoofing by attacker.
+  condition: consider_raw_packet_communication and evt.type=socket and evt.arg[0]=AF_PACKET and container and not proc.name in (user_known_raw_packet_binaries)
+  output: Raw packet was created in a container (user=%user.name command=%proc.cmdline container_id=%container.id container_name=%container.name image=%container.image.repository:%container.image.tag)
```

Comment:
can you add `socket_info=%evt.args` in the output section? Let's not have the `RAW_SOCK` in the condition this time. With the help of the output, we can identify the False Positive easily.

Reply:
And after looking into the ARP packet structure, I think `SOCK_DGRAM` is also able to spoof too.

Reply:
Thank you for checking it. I modified

### Output Example
```
{"output":"21:27:20.711269362: Notice Packet socket was created in a container (user=root command=socktest socket_info=domain=17(AF_PACKET) type=3 proto=1544  container_id=577c694601c8 container_name=k8s_ubuntu_ubuntu-9f765598b-jxwcd_default_e241e25f-1523-11ea-93b8-42010a80001b_1 image=ubuntu:latest) k8s.ns=default k8s.pod=ubuntu-9f765598b-jxwcd container=577c694601c8 k8s.ns=default k8s.pod=ubuntu-9f765598b-jxwcd container=577c694601c8 k8s.ns=default k8s.pod=ubuntu-9f765598b-jxwcd container=577c694601c8","priority":"Notice","rule":"Packet socket created in container","time":"2019-12-03T21:27:20.711269362Z", "output_fields": {"container.id":"577c694601c8","container.image.repository":"ubuntu","container.image.tag":"latest","container.name":"k8s_ubuntu_ubuntu-9f765598b-jxwcd_default_e241e25f-1523-11ea-93b8-42010a80001b_1","evt.args":"domain=17(AF_PACKET) type=3 proto=1544 ","evt.time":1575408440711269362,"k8s.ns.name":"default","k8s.pod.name":"ubuntu-9f765598b-jxwcd","proc.cmdline":"socktest","user.name":"root"}}
```